index.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express')
 const path = require('path')
-const boyParser = require('body-parser')
```

Comment:
you should rename boyParse to bodyParser in this line also 
https://github.com/mluukkai/example_app/blob/2d84be6a9a80710aaae7fb1db71d5fbe1e52add2/index.js#L11